compiler/modules/front.syntax/ast.bal

```diff
@@ -248,6 +250,7 @@ public type CheckingExpr record {|
     // *PositionFields
     Position startPos;
     Position endPos;
+    Position opPos;
```

Comment:
Is this the position of the `check` keyword? If so, then `opPos` is not a good name.  Use the same word here as for `error`. Maybe `kwPos`.

Reply:
fixed